schemas/preferences.sublime-settings.json

```diff
@@ -426,14 +430,21 @@
       "type": "boolean",
       "default": true
     },
+    "draw_shadows": {
+      "markdownDescription": "A boolean that can disable the shadow effect when any line is longer that the window.",
```

Comment:
typo: ... longer that ... -> ... longer than ...

Reply:
nice catch :D